templates/partials/modal_assign.haml

```diff
@@ -8,8 +8,9 @@
       %span{ ng-bind-html:"prompt" }
     .form-group
       %label.control-label
-        - trans "Assign to"
-      %select.form-control{ ng-model:"fields.assignee", ng-options:"p.name for p in partners track by p.id" }
+        - trans "Assign to Partner/User"
+      %select.form-control{ ng-model:"fields.assignee", ng-options:"p.name for p in partners track by p.id" , ng-change:"refreshUserList()" }
+      %select.form-control{ ng-model:"fields.user", ng-options:"u.name for u in users track by u.id", ng-show:"users.length" }
```

Comment:
Addressed in https://github.com/praekelt/casepro/pull/69/commits/a1594f89811097c8320281c300ccbda1077b38a1